### ISSUE TITLE
Interesting but complex

### ISSUE THREAD

**Issue by MikeyBeez** [deepspaceghost/deme-app#1]
Thank you for writing this interesting software.  I would like to try to merge this code with my own virtual assistant.  I don't use a database yet, for example.  I'm having some difficulty following the flow of your code, however.  Do you think you could add some explanations to your readme?  Cheers, and thanks. M  And perhaps I can help you add some of my code to yours, if you are interested.

**Comment by deepspaceghost** [comment]
Thank you for taking a look at it. A fresh pair of eyes are quite invaluable.

Yes, of course! Would you be okay isolating a line or function in particular? Once I have a better idea of what would be more useful, I can then use it as a template for more thorough documentation in the README file.

I'm all for it, and I'm willing to help where I can. I'd appreciate the opportunity to incorporate your code into my own, and I'm curious to see how my code can be of assistance. Thanks for reaching out!

**Comment by MikeyBeez** [comment]
I think I've figured out some of it.  I use Gitkraken to look at early commits, and it was the discord module that threw me.  Right now I'm looking at getting a Parlai model integrated.  I finally got my Tesla k-80 working again, and I don't want to pay for GPT-3 time.  So I need to find a model that doesn't suck too bad.  Then I'll be back to look into your database integration.  I've been learning Docker; so I do want to add a DB soon -- probably Mongo, and I'll do it as a microservice.  Nevertheless, I'm very curious about the way you are using your database.  It looked to me as though you use it a lot in a lot of places.  Cheers for now!  And thank you.  M
